src/posts/2023-11-06-cilium-certified-associate/index.md

```diff
@@ -0,0 +1,11 @@
+---
+date: '2023-11-06T12:33:00.000Z'
+title: 'COMING SOON! Cilium Certified Associate (CCA)'
+ogImage: ogimage.png
+ogSummary: 'The Cloud Native Computer Foundation and Linux Foundation Training and Certification announce the new Cilium Certified Associate (CCA) certification designed for platform and cloud engineers'
```

Comment:
```suggestion
ogSummary: 'The Cloud Native Computing Foundation and Linux Foundation Training and Certification announce the new Cilium Certified Associate (CCA) certification designed for platform and cloud engineers'
```

Reply:
Ah, good catch, it must have been my vscode autocomplete feature that caused this.